docs/first-use/query.txt

```diff
@@ -0,0 +1,211 @@
+
+:ref:`Open up the admin UI <import-test-data>`.
+
+Select the *Console* tab from the left-hand navigation menu.
+
+You should see something like this:
+
+.. image:: admin-ui-console.png
+
+In the previous document, we imported some tweets from the public Twitter
+timeline.
+
+If you want know more about the ``tweets`` table, select the *Tables* tab from
+the left-hand navigation menu. This screen will show you the table `schema`_.
+
+.. _schema: https://crate.io/docs/crate/reference/en/latest/general/ddl/create-table.html#schemas
+
+For an example query, why don't we filter those tweets so that we're only
+looking at tweets from people with more than 100 followers. We can do that with
+a query like so:
+
+.. code-block:: psql
+
+    SELECT *
+      FROM tweets
+     WHERE account_user['followers_count'] > 100
+     LIMIT 100;
+
+The ``tweets`` table has a column named ``account_user``. The ``account_user``
+is an `object`_, and can be queried into by specifying object attributes. In
+the query above, we're matching rows where the ``followers_count`` attribute of
+this object is more than ``100``.
+
+.. _object: https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html#object
+
+Type this query into the admin UI query console. Then select *EXECUTE QUERY*.
+
+You should see something like this:
+
+.. image:: admin-ui-results.png
+
+CrateDB's SQL support includes distributed `aggregation`_, `scalar`_ functions
+and `geospatial`_ support for more complex queries and data analysis. CrateDB
+also comes with `fulltext search`_.
+
+.. _aggregation: https://crate.io/docs/stable/sql/aggregation.html
+.. _scalar: https://crate.io/docs/stable/sql/scalar.html
+.. _geospatial: https://crate.io/docs/stable/sql/data_types.html#geo-point
+.. _fulltext search: https://crate.io/docs/crate/reference/en/latest/general/dql/fulltext.html
+
+.. SEEALSO::
+
+   Consult `the CrateDB query reference`_ for more information.
+
+.. _the CrateDB query reference: https://crate.io/docs/crate/reference/en/latest/general/dql/index.html
+
+Taking It Further
+=================
+
+Before we take a look at client libraries for CrateDB, let's take a quick look
+at two additional ways you can query CrateDB.
+
+The CrateDB Shell
+-----------------
+
+The CrateDB Shell (aka `Crash`_) is a command-line shell that comes with CrateDB.
+
+.. _Crash: https://crate.io/docs/clients/crash/en/latest/
+
+Start the shell like so:
+
+.. code-block:: sh
+
+   sh$ ./bin/crash
+
+First let's connect to a running node by typing:
+
+.. code-block:: psql
+
+    \connect localhost:4200;
+
+Hit :kbd:`Enter` and you will be connected to CrateDB running on ``localhost``.
+
+From here, you can execute queries against CrateDB by typing them and hitting
+:kbd:`Enter`.
+
+Let's try something new.
+
+Type out the following query:
+
+.. code-block:: psql
+
+     SELECT account_user['followers_count']
+       FROM tweets
+   ORDER BY account_user['followers_count'] DESC
+      LIMIT 10;
```

Comment:
pls fix indention

Reply:
the style I am using for SQL works like this:

```
                  CLAUSE parameters
ANOTHER CLAUSE more parameters
          3RD CLAUSE params
```

where the lines are indented so that the start of the parameters lines up